src/oncall/healthcheck.py

```diff
@@ -29,11 +32,13 @@ def on_get(self, req, resp):
                 cursor.close()
                 connection.close()
             except:
```

Comment:
While we're here, can we replace these bare excepts with `except Exception`, and in the case below `except IOError` ? (Bare excepts can have some unintentional behavior like silencing sys.exit() )

Reply:
changed